web_programming/currency_converter.py

```diff
@@ -173,12 +174,12 @@
 
 
 def convert_currency(
-    from_: str = "USD", to: str = "INR", amount: float = 1.0, api_key: str = API_KEY
+    from="USD", to="INR", amount=1.0, api_key=API_KEY
```

Comment:
Why _remove_ type hints?!?

Isn't `from` a reserved word in Python?!?

Reply:
Oh I misunderstood how Mypy works. And sorry didn't know `from` was reserved. I will fix both.

--- 

I know this works:

```python
x = 123 # inferred as int
x = '456'
# error because of changing type
```

And thought it applied here too

```python
def greet(v=123):
    print(v)


greet(456)

greet("456")
# no error
```
Yet no error is shown :(

Unless I add the explicit type.

```python
def greet(v: int =123):
    print(v)


greet(456)

greet("456")
# error...
```